plugin/pkg/scheduler/scheduler.go

```diff
@@ -124,7 +124,9 @@ func (s *Scheduler) scheduleOne() {
 	}
 
 	go func() {
-		defer metrics.E2eSchedulingLatency.Observe(metrics.SinceInMicroseconds(start))
+		defer func() {		
```

Comment:
because the defer function parameter is calculated first, it is just a rule, not code optimization